lib/Controller/RetrieveMetadataApiController.php

```diff
@@ -0,0 +1,48 @@
+<?php
+/**
+ * ownCloud - mediametadata
+ *
+ * This file is licensed under the Affero General Public License version 3 or
+ * later. See the COPYING file.
+ *
+ * @author Jalpreet Singh Nanda (:imjalpreet) <jalpreetnanda@gmail.com>
+ * @copyright Jalpreet Singh Nanda (:imjalpreet) 2016
+ */
+
+namespace OCA\MediaMetadata\Controller;
+
+
+use OCA\MediaMetadata\Services\RetrieveMetadata;
+use OCP\AppFramework\ApiController;
+
+/**
+ * Class RetrieveMetadataApiController
+ *
+ * @package OCA\MediaMetadata\Controller
+ */
+class RetrieveMetadataApiController extends ApiController {
```

Comment:
And the name needs to be MetadataController if you want to match the route.

https://docs.nextcloud.com/server/10/developer_manual/app/routes.html